app/vulncode_app/views.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.http import HttpResponse
+from django.shortcuts import render
+
+import vulncode_app.api_data as api
+
+def product(request, product):
```

Comment:
You are using `package` in the api_data module and `product` here, any particular reason?